portal-web/test/functional/com/liferay/portalweb/macros/User.macro

```diff
@@ -1319,7 +1319,11 @@ definition {
 	macro editAddressCP {
 		UserNavigator.gotoContact();
 
-		LexiconEntry.gotoEllipsisMenuItem(menuItem = "Edit");
+		Click.waitForMenuToggleJSClick(locator1 = "Icon#VERTICAL_ELLIPSIS_BUTTON");
```

Comment:
Please use the exist macro instead, you can find it via this locator you used. Thanks in advance.